crates/mako/src/visitors/react.rs

```diff
@@ -123,7 +123,7 @@ prevRefreshSig = self.$RefreshSig$;
 self.$RefreshReg$ = (type, id) => {
   RefreshRuntime.register(type, module.id + id);
 };
-self.$RefreshSig$ = RefreshRuntime.createSignatureFunctionForTransform;
+self.$RefreshSig$ = () => " %exports%";
```

Comment:
the function should be `()=> () => " %exports%";`
but why hardcode as `" %exports%";`

what about we guess as sig from class's name or something else

Reply:
Use `RefreshRuntime.createSignatureFunctionForTransform` instead of hardcoding incorrectly.